snowpack/src/rewrite-imports.ts

```diff
@@ -28,7 +27,7 @@ export async function scanCodeImportsExports(code: string): Promise<any[]> {
 
 export async function transformEsmImports(
   _code: string,
-  replaceImport: (specifier: string) => string,
+  replaceImport: (specifier: string) => string | Promise<string>,
```

Comment:
WDYT about just always returning `Promise<string>` instead? Unless that’s not possible for some reason